### PR TITLE
add two scripts for report generate

### PR DESCRIPTION
**Pull request by ESI-SYD**
1. process.py for generating a report from specified two directories of regular refresh test results.
2. parse_inductor_ipex_log.py for generating a report of absolute data about IPEX vs Inductor test.

### CODE REVIEW

File: scripts/modelbench/process.py [hunk 1]
```diff
@@ -0,0 +1,249 @@
+"""process.py
```
[the remark this answers is not in the document]
Comment (ESI-SYD) [reply scripts/modelbench/process.py:1]: yes, renamed.

File: scripts/modelbench/process.py [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+"""
+
+
+import argparse
+import pandas as pd
+from pandas import ExcelWriter
+import os
+
+parser = argparse.ArgumentParser(description="Generate report from recently two round inductor_log")
+parser.add_argument('-l','--lr',type=str,help='last round log file')
```
[the remark this answers is not in the document]
Comment (ESI-SYD) [reply scripts/modelbench/process.py:18]: done

File: scripts/modelbench/process.py [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    header = {"A": '', "B": args.tr, "C": '', "D": '',"E": '', "F": args.lr, "G": '', "H": '',"I": '',"J": 'Result Comp',"K": '',"L": ''}
+    h = pd.DataFrame(header, index=[0])
+    h.to_excel(writer, sheet_name='Single-Socket Multi-threads', index=False,startrow=0,header=False)
+    h.to_excel(writer, sheet_name='Single-Socket Single-thread', index=False,startrow=0,header=False)
+    
+    # update cmp data
+    mt=process_thread('multi_threads_cf_logs')
+    st=process_thread('single_thread_cf_logs')
+
+    mt[['name','batch_size_x','speedup_x']].to_excel(writer, sheet_name='Single-Socket Multi-threads', index=False, startrow=1, startcol=0)
```
[the remark this answers is not in the document]
Comment (ESI-SYD) [reply scripts/modelbench/process.py:211]: done

File: scripts/modelbench/process.py [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+def generate_report(lr,tr):
+    with ExcelWriter('Inductor Dashboard Regression Check '+tr+'.xlsx') as writer:
+        update_summary(writer,lr,tr)
+        update_swinfo(writer)
+        update_failures(writer)
+        update_details(writer)
+
+if __name__ == '__main__':
+    generate_report(args.lr, args.tr)
```
[the remark this answers is not in the document]
Comment (ESI-SYD) [reply scripts/modelbench/process.py:249]: done

**Review by chuanqi129 — APPROVED** [review]

**Pull request merged**